readthedocs/projects/models.py

```diff
@@ -524,6 +524,14 @@ class Project(models.Model):
     featured = models.BooleanField(_("Featured"), default=False)
 
     skip = models.BooleanField(_("Skip (disable) building this project"), default=False)
+    n_consecutive_failed_builds = models.BooleanField(
```

Comment:
Maybe we can make this a more generic name so it can be used for disabling projects based on other rules (`builds_disabled` or `builds_enabled`)? The notification would show the reason to the user.